s3watcher/lambda/lib/Transfer.js

```diff
@@ -59,7 +59,12 @@ module.exports = {
             log("download", s3Object);
 
             return S3Helper.getS3Object(s3Object).flatMap(function(data){
-                log("upload", upload);
+                log("upload", {
+                    size: upload.size,
+                    filename: upload.params.filename,
+                    uploadedBy: upload.params.uploadedBy,
+                    stage: upload.params.stage
+                });
```

Comment:
`upload` object is from [here](https://github.com/guardian/grid/blob/master/s3watcher/lambda/lib/Upload.js#L23-L34).